scripts/postNpmInstall.js

```diff
@@ -0,0 +1,10 @@
+// Remove the webpack-dev-server grunt task since we do not use it
+var fs = require('fs');
+var path = require('path');
+
+var taskFile = path.join(__dirname, '..', 'node_modules', 'grunt-webpack', 'tasks', 'webpack-dev-server.js');
```

Comment:
Does `require.resolve('grunt-webpack')` work here?  That would be a more resilient method of finding it's path.